### PR TITLE
ARROW-6505: [Website] Add new committers

### PR DESCRIPTION
**Pull request by mrkn**
I'd like to add new committers on the list.

### CODE REVIEW

File: committers.html [hunk 1]
```diff
@@ -251,6 +251,24 @@ <h2>Committers</h2>
 <td>TBD</td>
 <td>Dremio</td>
 </tr>
+<tr>
+<td>Ben Kietzman</td>
+<td>Committer</td>
+<td>bkietz</td>
+<td></td>
```
Comment (mrkn) [inline committers.html]: @bkietz Could you tell me your affiliation?

File: committers.html [hunk 2]
```diff
@@ -251,6 +251,24 @@ <h2>Committers</h2>
...
+<td>Kenta Murata</td>
+<td>Committer</td>
+<td>mrkn</td>
+<td>Speee, Inc.</td>
+</tr>
+<tr>
+<td>Neal Richardson</td>
+<td>Committer</td>
+<td>npr</td>
+<td></td>
```
Comment (mrkn) [inline committers.html]: @nealrichardson Could you tell me your affiliation?

**Comment by wesm** [comment]
I added the missing affiliations. +1

**Comment by mrkn** [comment]
Thanks, @wesm!

**Pull request merged**